File: sumCube.js
function sumCubes(n){
    let count = 1
    let arr = []
    while (count <=n){
      arr.push(count)
      count++
    }
    let cubeArr =[]
    for (let i = 0; i < arr.length; i++){
      cubeArr.push(Math.pow(arr[i], 3))
    }
    let summedArr = cubeArr.reduce((a,b) => a + b, 0)
    return summedArr
  }

File: sumCube.py
def sum_cubes(n):
    count = 1
    arr = []
    while count <= n:
        arr.append(count)
        count += 1
    cubeArr = []
    for x in arr:
        cubeArr.append(x ** 3)
    summed = sum(cubeArr)
    return summed